lib/perfil/profileConf.dart

```diff
@@ -0,0 +1,275 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/cupertino.dart';
```

Comment:
```suggestion
import 'package:flutter/cupertino.dart';
```
Esto creo que es para iOS y si solo vamos a hacer Android no sé si es bueno utilizarlo. Creo que también lo tenias en la de perfil.

Reply:
Lo reviso ahora